sql/plan/grant.go

```diff
@@ -664,33 +666,30 @@ func (n *GrantRole) CheckPrivileges(ctx *sql.Context, opChecker sql.PrivilegedOp
 	//TODO: only active roles may be assigned if the SUPER privilege is not held
 	mysqlDb := n.MySQLDb.(*mysql_db.MySQLDb)
 	client := ctx.Session.Client()
-	user := mysqlDb.GetUser(client.User, client.Address, false)
+
+	// TODO: Locking
```

Comment:
Is there more locking needed here beyond what the reader encapsulates? I didn't understand concretely what this TODO was recommending.

Reply:
This was a stale comment from before we had locking! Good catch :D.